test/rest_api/test_compaction_task.py

```diff
@@ -36,9 +36,9 @@ def check_compaction_task(cql, this_dc, rest_api, run_compaction, compaction_typ
 
 def test_major_keyspace_compaction_task(cql, this_dc, rest_api):
     # keyspace major compaction
-    check_compaction_task(cql, this_dc, rest_api, lambda keyspace, _: rest_api.send("POST", f"storage_service/keyspace_compaction/{keyspace}"), "major compaction", 2)
+    check_compaction_task(cql, this_dc, rest_api, lambda keyspace, _: rest_api.send("POST", f"storage_service/keyspace_compaction/{keyspace}"), "major compaction", 3)
```

Comment:
How does s/2/3/ extend the test to test major compaction tasks?
Maybe use a named variable for this parameter to make it clearer.